### PR TITLE
Docker Workflow

### PR DESCRIPTION
**Pull request by da-Kai**
Build and Release Docker images on tag events.

continuation of #2358

### CODE REVIEW

**Comment by da-Kai** [comment]
@sfeilmeier is there a OpenEMS Docker Hub account?

**Comment by sfeilmeier** [comment]
> @sfeilmeier is there a OpenEMS Docker Hub account?

No, not yet. Feel free to create one.

**Comment by da-Kai** [comment]
> No, not yet. Feel free to create one.

https://hub.docker.com/u/openems already exists :/

**Review by sfeilmeier — CHANGES_REQUESTED** [review]
Looks good. I still need somebody else to do a detailed review first. Maybe Manoj?

**Comment by manojniit** [comment]
> Looks good. I still need somebody else to do a detailed review first. Maybe Manoj?


Everything is working properly. I checked it.

File: tools/docker/openems-edge/config.d/Controller/Api/Backend/28628d01-a978-4328-9a8f-b9c7751ed2d2.config
Comment (sfeilmeier) [inline tools/docker/openems-edge/config.d/Controller/Api/Backend/28628d01-a978-4328-9a8f-b9c7751ed2d2.config:1]: Gitpod is using these files -> https://github.com/OpenEMS/openems/blob/develop/.gitpod.yml#L17

I did not test it, but this PR would (again) break Gitpod, right?

**Review by sfeilmeier — COMMENTED** [review]
Thank you!

Special acknowledgement for the ASCII-Art 😍🚀

**Pull request merged**